src/USB.ml

```diff
@@ -69,12 +68,12 @@ let transfer_error_message = function
 let () =
   Printexc.register_printer
     (function
-       | Error(err, fun_name) ->
-           Some(Printf.sprintf "USB error: '%s' failed: %s" fun_name (error_message err))
-       | Transfer(err, fun_name) ->
-           Some(Printf.sprintf "USB transfer error: '%s' failed: %s" fun_name (transfer_error_message err))
-       | exn ->
-           None)
+      | Error(err, fun_name) ->
+        Some(Printf.sprintf "USB error: '%s' failed: %s" fun_name (error_message err))
+      | Transfer(err, fun_name) ->
+        Some(Printf.sprintf "USB transfer error: '%s' failed: %s" fun_name (transfer_error_message err))
+      | e ->
+        Some (Printexc.to_string e))
```

Comment:
It should really be [None] here, [Printexc] takes care of trying the other registered printers. This line can actually create an infinite loop